<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFormDoc</class>
 <widget class="QWidget" name="QFormDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>设置字体</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销编辑操作</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string> 重复</string>
   </property>
   <property name="toolTip">
    <string>重复编辑操作</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
